hack/ostree_tag.sh

```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
 if pkg-config ostree-1 2> /dev/null ; then
-	echo ostree
-else
-	echo containers_image_ostree_stub
+	echo ostree ostree_repos
```

Comment:
This will need to be updated to use `containers_image_ostree` instead of `ostree_repos`.